apps/coscrad-frontend/src/utils/generic-components/presenters/get-text-item-by-language-code.ts

```diff
@@ -0,0 +1,8 @@
+import { IMultilingualText, IMultilingualTextItem, LanguageCode } from '@coscrad/api-interfaces';
+
+export const getTextItemByLanguageCode = (
```

Comment:
It's a shame we can't just share the multilingual text class with the front-end. Let's hold off, but we should think about this at some point.